pathz/index.md

```diff
@@ -57,6 +58,15 @@ with the expected to be deployed AuthroizationPolicy.
 
 ## User Experiences
 
+### gNMI interaction with Subscribe, Set or Get
+
+When a client makes a request to gNMI with a system with a pathz policy
+installed, the pathz policy is evaluated against the paths requested. The
+if there is no explicit permit to the path requested the client the RPC must
+be rejected. This keeps the gNMI server from being potentially DOS'ed by
+clients requesting top level paths which then have to be recursed for all
```

Comment:
Could you please provide an example? 

If the client subscribes to `/interfaces/`, but the pathz has a permit rule only to `/interfaces/interface[name=eth0]`, is this request getting denied?

Reply:
yes

Reply:
this was described at 179 in the authorization-README.md